index.html

```diff
@@ -736,27 +797,25 @@
           Emits an the event initialized with the event name specified by the <code>eventName</code> argument and data specified by the <code>payload</code> argument.
         </p>
         <p class="ednote">
-          This method in't actually needed to be exposed by the <a>ExposedThing</a> object since it is usually provided by the runtime.
+          This method isn't actually needed to be exposed by the <a>ExposedThing</a> object since it is usually provided by the runtime.
```

Comment:
You're right that we need a method to notify observers, and that can be done as emitting an Event. 

But the note here just says the Scripting API does not need to explicitly expose this call, because emitting an event is always implemented in any JS runtime. So I am not sure why do we need to expose it this way. In most implementations, e.g. in Node.js we already have [emit()](https://nodejs.org/api/events.html#events_emitter_emit_eventname_args), now we will have an additional `emitEvent()` which does the same.

Do we differentiate between WoT events and runtime events - do we need to?